public/svgs/frost/list-large.svg

```diff
@@ -1,6 +1,13 @@
-<svg viewBox="0 0 15 15" >
-	<g id="Layer_1_1_" class="view-size-svg">
-		<rect x="0" y="0" width="15" height="7"/>
-		<rect x="0" y="8" width="15" height="7"/>
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I've always understood it is the proper way to begin an SVG file.  Inline-svg code doesn't need this, but all the examples I've ever seen on Mozilla or W3C have it in standalone files.  I assume it tells some parses the XML rules to use to parse it.  Certain it can't hurt anything.